specification/sql/resource-manager/Microsoft.Sql/2014-04-01/queries.json

```diff
@@ -0,0 +1,347 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database",
+    "description": "Provides create, read, update and delete functionality for Azure SQL Database resources including servers, databases, elastic pools, recommendations, operations, and usage metrics.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/topQueries": {
```

Comment:
A few Naïve questions to help my understanding - What categorizes a query into top queries? Is it the number of executions? Do you already have an API for listing all queries under a database? 
If yes, you can consider adding a property to response of /queries to indicate whether a query is a top query or not. And then to just list top queries, customers can use $filter on that property? 
All this is based on my limited knowledge of the scenario.